daemon/archive.go

```diff
@@ -5,6 +5,7 @@ import (
 	"os"
 	"strings"
 
+	"github.com/docker/docker/api/errdefs"
```

Comment:
Wondering if we're now making separation between API and internals worse?

Reply:
This is the daemon package.